frame/democracy/src/lib.rs

```diff
@@ -1676,19 +1932,117 @@ impl<T: Trait> Module<T> {
 	}
 
 	/// Current era is ending; we should finish up any proposals.
-	fn begin_block(now: T::BlockNumber) -> DispatchResult {
+	///
+	///
+	/// # <weight>
+	/// If a referendum is launched or maturing take full block weight. Otherwise:
+	/// - Complexity: `O(R)` where `R` is the number of unbaked referenda.
+	/// - Db reads: `LastTabledWasExternal`, `NextExternal`, `PublicProps`, `account`,
+	///   `ReferendumCount`, `LowestUnbaked`
+	/// - Db writes: `PublicProps`, `account`, `ReferendumCount`, `DepositOf`, `ReferendumInfoOf`
+	/// - Db reads per R: `DepositOf`, `ReferendumInfoOf`
+	/// # </weight>
```

Comment:
I forgot to add the raw weight values here. Can you do that for each of the benchmarks written for `on_initialize`?

Reply:
You mean for other pallet ? yes I'll take a look on monday in another PR

Reply:
Not for other pallet, just for this. You will see I added comments to your code with all the raw benchmarking data, but I did not do it here. Would be good to have the raw data here if we are using it for weight calculations

Reply:
Yes done